CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- Include all object classes when using archiver. (#42)
+
```

Comment:
Generally, should I use "Unreleased" or the next version (assuming its 7.4.2)?

Reply:
"Unreleased" if the next release is not defined yet.

In this case, this fix seems important and safe enough that we should do a 7.4.2 release after running through the internal staging and testing processes. Do you agree?